draft-hoffman-dns-over-http-latest.mkd

```diff
@@ -298,20 +304,20 @@ query used HTTP ETag headers as described in {{RFC7232}}.
 For example, assume that the server's PREFIX is:
 
 ~~~
-   https://dnsserver.example.net/
+   https://dnsserver.example.net/api1
```

Comment:
may as well do `https://dnsserver.example.net/api/v1/`, and it *really* needs to end with a "/", if one is required.  Additionally, we need to talk about checking for well-formedness after creating the URI from the template.